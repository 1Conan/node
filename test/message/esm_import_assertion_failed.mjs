import '../common/index.mjs';
import 'data:text/javascript,export{}' assert {type:'json'};
