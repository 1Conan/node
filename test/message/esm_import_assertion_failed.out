node:internal/errors:*
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_IMPORT_ASSERTION_TYPE_FAILED]: Module "data:text/javascript,export{}" is not of type "json"
    at new NodeError (node:internal/errors:*:*)
    at handleInvalidType (node:internal/modules/esm/assert:*:*)
    at validateAssertions (node:internal/modules/esm/assert:*:*)
    at defaultLoad (node:internal/modules/esm/load:*:*)
    at ESMLoader.load (node:internal/modules/esm/loader:*:*)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:*:*)
    at new ModuleJob (node:internal/modules/esm/module_job:*:*)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:*:*)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:*:*)
    at async ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:*:*) {
  code: 'ERR_IMPORT_ASSERTION_TYPE_FAILED'
}
Node.js *
